apps/meteor/client/views/room/composer/messageBox/MessageComposerHints/EditingComposerHint.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Icon, Tooltip } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+const EditingComposerHint = (): ReactElement => {
+	const t = useTranslation();
+	return (
+		<>
+			<Box display='flex' flexDirection='row' justifyContent='space-between' alignItems='end'>
+				<Tooltip variation='light' fontScale='c2'>
+					<Icon name='pencil' size='x16' marginInlineEnd='x5' />
+					{t('Editing_Message')}
+				</Tooltip>
+				<Box color='font-hint'>
+					<Box display='inline' fontWeight='c2'>
+						esc
+					</Box>{' '}
+					to {t('Cancel')}
+					{' · '}
+					<Box display='inline' fontWeight='c2'>
+						enter
+					</Box>{' '}
+					to {t('Save')}
```

Comment:
This entire string can be a translation key too... check it out [PlanCardTrial.tsx](https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/client/views/admin/subscription/components/cards/PlanCard/PlanCardTrial.tsx)

There are other examples of this feature too

Reply:
I implemented it but probably my version is better if these translations are manually written in the json files for other languages. It is because 'esc' and 'enter' I guess is same for all keyboards round the world and 'cancel' and 'save' are already translated words. Correct me if I am wrong due my lack of knowledge